web/controllers/caseload/user_controller.ex

```diff
@@ -22,6 +22,8 @@ defmodule Healthlocker.Caseload.UserController do
   end
 
   def show(conn, %{"id" => id}) do
```

Comment:
Actually instead of doing the redirect here, couldn't we just change the original link to point to the `caseload_user_room_path`? It feels a bit strange to just have an action which always redirects.